src/main/java/appeng/client/render/blocks/RenderBlockSkyChest.java

```diff
@@ -91,25 +95,26 @@ public void renderTile( AEBaseBlock block, AEBaseTile tile, Tessellator tess, do
 			return;
 		}
 
-		TileSkyChest skyChest = (TileSkyChest) tile;
+		final TileSkyChest skyChest = (TileSkyChest) tile;
 
 		if( !skyChest.hasWorldObj() )
 		{
 			return;
 		}
 
-		GL11.glEnable( 32826 /* GL_RESCALE_NORMAL_EXT */ );
+		GL11.glPushAttrib( GL11.GL_POLYGON_BIT );
+		GL11.glEnable( GL12.GL_RESCALE_NORMAL );
 		GL11.glColor4f( 1.0F, 1.0F, 1.0F, 1.0F );
 
 		ResourceLocation loc;
 
 		if( tile.getWorldObj().getBlockMetadata( tile.xCoord, tile.yCoord, tile.zCoord ) == 1 )
 		{
-			loc = new ResourceLocation( "appliedenergistics2", "textures/models/skyblockchest.png" );
+			loc = SKY_BLOCK_CHEST;
 		}
 		else
 		{
-			loc = new ResourceLocation( "appliedenergistics2", "textures/models/skychest.png" );
+			loc = SKY_STONE_CHEST;
 		}
 
```

Comment:
should be able to be moved here 
also best-practive to use `{...}` to indicate the mirror